pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1550,9 +1550,10 @@ func (m *kubeGenericRuntimeManager) doBackOff(pod *v1.Pod, container *v1.Contain
 			m.recorder.Eventf(containerRef, v1.EventTypeWarning, events.BackOffStartContainer,
 				fmt.Sprintf("Back-off restarting failed container %s in pod %s", container.Name, format.Pod(pod)))
 		}
-		err := fmt.Errorf("back-off %s restarting failed container=%s pod=%s", backOff.Get(key), container.Name, format.Pod(pod))
+		backoff := backOff.Get(key)
```

Comment:
I think what you want here is the remaining backoff, which would be
```go
remainingBackoff := backoff - time.Since(ts)
```

Although, I wonder if it's better to use an absolute timestamp in case the rest of the SyncPod work is slow? I.e.
```
backoffTime := ts.Add(backoff)
```